spec/acceptance/00_pkg_rpm_custom_scriptlets_spec.rb

```diff
@@ -37,28 +37,28 @@
     comment '...default preun postun scriptlets call simp_rpm_helper with correct arguments'
     expected_simp_rpm_helper_scriptlets = scriptlet_label_map.select{|k,v| %w(preun postun).include? v }
     expected_simp_rpm_helper_scriptlets.each do |rpm_label, simp_helper_label|
-      expected = <<EOM
-if [ -x /usr/local/sbin/simp_rpm_helper ] ; then
-  /usr/local/sbin/simp_rpm_helper --rpm_dir=/usr/share/simp/modules/testpackage --rpm_section='#{simp_helper_label}' --rpm_status=$1
-fi
-EOM
+      expected = <<~EOM
+        if [ -x /usr/local/sbin/simp_rpm_helper ] ; then
```

Comment:
Does this file need a `retab`?

Reply:
I do not see any issues with the spacing.  Can you clarify?